app/Http/Controllers/Users/MessagesController.php

```diff
@@ -15,10 +15,13 @@ public function postMessage(Request $request, string $discordId, CreateMessage $
 
         $payload = $this->validate($request, [
             'discord_id' => ['required','exists:users'],
```

Comment:
You need to remember that this `exists:` uses the eloquent under the hood. It would be better if you validate if the user exists in the same query where you save the message, using the MessageRepository.